packages/blog/src/BlogPostTeaser.jsx

```diff
@@ -1,6 +1,10 @@
 import './BlogPostTeaser.css';
 import {fetchList, fetchPostByCategorySlug} from './Data';
 import {Link, useParams} from "react-router-dom";
+import { fetchList } from './Data';
```

Comment:
Hey @lilit-harutyunyan! Danke für's Helfen! Aber die Zeilen 4 und 5 sind doppelt, `fetchList` und `Link` werden schon in den Zeilen 2 und 3 importiert. Z4-5 können also einfach entfernt werden.